crates/solidity/outputs/npm/package/syntax/nodes/index.js

```diff
@@ -6,6 +6,7 @@
 
 const generated = require("../../generated");
 
+module.exports.Cursor = generated.Cursor;
```

Comment:
I suggest moving `Cursor` to `syntax/visitors` namespace to match the other export.

Reply:
I would rather have cursor and visitor in nodes, so I think I'll change the other export